### ISSUE TITLE
UVG dataset experiment options

### ISSUE THREAD

**Issue by hyo7** [haochen-rye/NeRV#11]
Thanks for sharing your research.

I'm trying to reproduce the Figure 7 graph of your paper(PSNR vs. BPP on UVG dataset), but I couldn’t find the appropriate experiment options. Could you tell me (C1,C2) for that result? (among Appendix A.1’s values)

Other options I've tried so far are as follows.
- Learning rate:
  op1. 5e-4 (paper 4.1)
  op2. 5e-4 x 6 (linear rule /w batch size 6)
- Up-scale factor: 5,3,2,2,2 (paper 4.1)
- Train epochs: 1500 epochs (paper 4.1)
- Warmup epochs:
  op1. 300 epochs (train code's default. train epochs * 0.2)
  op2. 30 epochs (paper 4.1)

**Comment by haochen-rye** [comment]
Although we train one NeRV model for all 7 videos in the paper, but train 7 small models for 7 videos separately should give you similar bpp-psnr performance, and greatly reduce your training time!

For learning rate, op2 should give you similar psnr with op1
Your up-scale factor looks good
To fit one NeRV to one video, the epoch for convergence might differ slightly (from 600 epochs to 1200 epochs).
For warmup epochs, either op1 or op2 should be fine

**Comment by haochen-rye** [comment]
For numbers in the paper, we use batch size 1 and learning rate 5e-4, to build NeRV with different model sizes, we change the FCdim (output channel for 1st FC) and expansion (at the beginning of ConvBlock), we tried `64_8, 128_4, 128_6, 128_8, 192_8` for `FCdim_expansion` setups.

**Comment by gatorgirl-choi** [comment]
Thank you for sharing your work!
I am also interested in reproducing Fig7 (uvg experiment).
I found the results in psnr_bpp_results.csv. Could you kindly share what pruning ratio(es) and quantization bit(s) have been used for the plot?

**Comment by haochen-rye** [comment]
For model weight quantization, we always use 8-bits.
For pruning, we use global pruning with sparsity 25%, 20%, and 15%, and choose the best one for different model sizes.

**Comment by hyo7** [comment]
Thanks for your reply! I will try again with the option in your comment.
